packages/contracts-bedrock/test/L1/StandardValidator.t.sol

```diff
@@ -118,567 +102,105 @@ contract StandardValidatorV180_Test is Test {
-                l1ERC721BridgeImpl: address(0xAE2AF01232a6c4a4d3012C5eC5b1b35059caF10d),
-                optimismMintableERC20FactoryImpl: address(0xE01efbeb1089D1d1dB9c6c8b135C934C0734c846),
-                disputeGameFactoryImpl: address(0xc641A33cab81C559F2bd4b21EA34C290E2440C2B),
-                mipsImpl: address(0x5fE03a12C1236F9C22Cb6479778DDAa4bce6299C),
-                anchorStateRegistryImpl: address(0x1B5CC028A4276597C607907F24E1AC05d3852cFC),
-                delayedWETHImpl: address(0x71e966Ae981d1ce531a7b6d23DC0f27B38409087)
-            }),
-            ISuperchainConfig(address(0x95703e0982140D16f8ebA6d158FccEde42f04a4C)),
-            address(0x5a0Aae59D09fccBdDb6C6CcEB07B7279367C3d2A), // l1PAOMultisig
-            address(0x5fE03a12C1236F9C22Cb6479778DDAa4bce6299C), // mips
-            address(0x9BA6e03D8B90dE867373Db8cF1A58d2F7F006b3A) // challenger
-        );
-
-        StandardValidatorV180.Input memory input = StandardValidatorV180.Input({
-            proxyAdmin: IProxyAdmin(address(0x543bA4AADBAb8f9025686Bd03993043599c6fB04)),
-            sysCfg: ISystemConfig(address(0x229047fed2591dbec1eF1118d64F7aF3dB9EB290)),
-            absolutePrestate: bytes32(0x03f89406817db1ed7fd8b31e13300444652cdb0b9c509a674de43483b2f83568),
-            l2ChainID: 10
-        });
-
-        // OP Mainnet has a different expected root than the default one, so we expect to see ANCHORP-40.
-        string memory errors = mainnetValidator.validate(input, true);
-        assertEq(errors, "PDDG-ANCHORP-40,PLDG-ANCHORP-40");
     }
 
-    /// @notice Tests that validation succeeds with valid inputs and mocked dependencies
-    function test_validate_allowFailureTrue_succeeds() public {
-        StandardValidatorV180.Input memory input = StandardValidatorV180.Input({
-            proxyAdmin: proxyAdmin,
-            sysCfg: systemConfig,
-            absolutePrestate: absolutePrestate,
-            l2ChainID: l2ChainID
-        });
-
-        // Mock all necessary calls for validation
-        _mockValidationCalls();
-
-        // Validate with allowFailure = true
-        string memory errors = validator.validate(input, true);
-        assertEq(errors, "");
+    // Helper function to assert error codes in validation
+    function assertErrorCode(string memory _errCode, string memory errCodes) internal pure {
+        assertEq(errCodes, _errCode);
     }
 
-    /// @notice Tests that validation reverts with error message when allowFailure is false
-    function test_validate_allowFailureFalse_reverts() public {
-        StandardValidatorV180.Input memory input = StandardValidatorV180.Input({
-            proxyAdmin: proxyAdmin,
-            sysCfg: systemConfig,
-            absolutePrestate: absolutePrestate,
-            l2ChainID: l2ChainID
-        });
-
-        _mockValidationCalls();
+    // All the mock helper functions moved here but kept exactly the same
```

Comment:
Since this comment won't make sense after merge, but thanks for adding it here
```suggestion
```

Reply:
Ditto here - I think this is an AI refactor artifact. I'll give it another manual look through and fix any other such issues I find.